src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1081,6 +1082,73 @@ private void ApplyTextDocumentChange(DocumentId documentId, SourceText newText)
             }
         }
 
+        protected override void ApplyDocumentInfoChanged(DocumentId id, DocumentInfo updatedInfo)
```

Comment:
i see.  so this is an override of an existing method that the workspace has?

Reply:
Yes.